### DISCUSSION TITLE
ESP32-C6

### DISCUSSION THREAD

**Discussion by frver0x** [espressif/arduino-esp32#8252] [Q&A] [answered]
Hello, i recently bought and esp32-C6 and to my great surprise it is not compatible with the arduino ide, do you know if it will soon be compatible?
Thanks

**Comment by mrengineer7777** [comment]
C6 support should be in release 3.0.0, which will be sometime this year.  No idea when it will be ready.

**Comment by VojtechBartoska** [comment, marked as answer]
support of C6 is tracked here: https://github.com/espressif/arduino-esp32/issues/7713
